cmd/device-worker/main.go

```diff
@@ -74,7 +74,7 @@ func main() {
 	// Listen on the provided socket address.
 	l, err := net.Listen("unix", r.GetAddress())
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal("Cannot start listening on %s err: ", r.GetAddress(), err)
```

Comment:
I think that should be `Fatalf`.

Reply:
I think there will be a mismatch between the number of arguments in the format string to the number of provided arguments.
Perhaps change to:
```
log.Fatalf("Cannot start listening on %s err: %v", r.GetAddress(), err)
```

Reply:
Fixed sorry!